Cargo.toml

```diff
@@ -21,3 +21,10 @@ maintenance = { status = "passively-maintained" }
 base64 = "0.12"
 quoted_printable = "0.4.2"
 charset = "0.1.1"
+
+[dev-dependencies]
+criterion = "0.3"
```

Comment:
While I appreciate the performance testing, I don't want to land anything that adds extra dependencies or dev-dependencies to mailparse. I feel pretty strongly about keeping the dependency graph/trust chain small, and anything not absolutely essential shouldn't be here. So I'll happily merge the two commits that actually fix the issue (with my other review comment addressed) but I'd like to skip this benchmarking stuff.

Reply:
Sure, no problem. I just figured this is a performance-critical crate, so didn't want to cause a regression in the (more likely) existing cases.